### DISCUSSION TITLE
Stop retrieving messages

### DISCUSSION THREAD

**Discussion by tecdaddy** [microsoft/mfcmapi#518] [Q&A]
I am having an issue that may be beyond MFCMAPI's scope. I had to create a new .pst file in Outlook and import the messages from the old .pst file into the new one. Now when I click Send/Receive Outlook is trying to retrieve every message (all the way back to 2010) from the pop3 server. I read that since Outlook 2010 the way it keeps up with which messages have already been downloaded is stored somewhere _inside_ the .pst file. The only program I have any experience with that can edit a .pst file is MFCMAPI. 

**TL;DR**
Do you know if I can use MFCMAPI to edit the .pst file to stop retrieving messages messages older than say the last 24hrs?
